rollup.config.js

```diff
@@ -0,0 +1,25 @@
+import nodeResolve from 'rollup-plugin-node-resolve';
```

Comment:
aint sure, but this plugin might not be needed here, we do not import any dependencies from the `node_modules` to be included in the bundle (`react` and `react-dom` are treated as external)

so would be nice to have it checked and possibly removed